Automattic/simplenote-electron
# Open new note upon creation

### Fix
https://github.com/Automattic/simplenote-electron/issues/1566

This PR updates the logi so that clicking new note not only creates that note but opens it. This was only a problem on smaller screens where only the note or the note list show.

### Test
1. Decrease the viewport of your browser or electron app so that it is single-paned
2. Click the new note icon
3. Did the new open so that you can edit?
4. Repeat 1 to 3 using production
5. Observe note does not open

### Review
Only one developer is required to review these changes, but anyone can perform the review.

### Release
`RELEASE-NOTES.txt` was updated with:

- Open new note automatically upon creation [1566](https://github.com/Automattic/simplenote-electron/issues/1566)
